dot/services_integration_test.go

```diff
@@ -39,6 +45,55 @@ func TestCreateStateService(t *testing.T) {
 	require.NotNil(t, stateSrvc)
 }
 
+func newStateServiceWOMock(t *testing.T) *state.Service {
```

Comment:
nit Using `Without` instead of `WO` would be nice

Reply:
Updated.